lib/components/admin/field-trip-details.js

```diff
@@ -73,9 +82,16 @@ class FieldTripDetails extends Component {
   }
 
   _setRequestDate = (evt) => {
-    const { dateFormat, intl, request, setRequestDate } = this.props
-    const convertedRequestDate = moment(evt.target.value).format(dateFormat)
-    setRequestDate(request, convertedRequestDate, intl)
+    const newDate = evt.target.value
+    // Only persist the date if it is valid (not an empty string)
+    if (newDate !== '') {
+      const { dateFormat, intl, request, setRequestDate } = this.props
+      const convertedRequestDate = moment(newDate).format(dateFormat)
+      setRequestDate(request, convertedRequestDate, intl)
+      this.setState({
+        travelDateUpdateCount: this.state.travelDateUpdateCount + 1
```

Comment:
Do we use this count for anything else than updating the key? If not, would using `uuid()` not make more sense? This is what we do elsewhere when we have this problem -- the library is already in the `yarn.lock`